src/pages/ProfileForm/Step1Form/Step1FormHeader.tsx

```diff
@@ -24,8 +24,10 @@ function Step1FormHeader({ isStep1 }: Step1FormHeaderProperties): JSX.Element {
           <>
             In order to begin using the platform you must have a Legal Entity
             Identifier (LEI) for your financial institution. Visit the{' '}
-            <Link href={gleifLink}>Global LEI Foundation (GLEIF)</Link> website
-            for more information on how to obtain an LEI.
+            <Link className='border-b-[1px] border-dotted' href={gleifLink}>
+              Global LEI Foundation (GLEIF)
+            </Link>{' '}
+            website for more information on how to obtain an LEI.
           </>
```

Comment:
I think when we change these fragments to `<p>` elements, the link styling takes care of itself - so wouldn't need to pass classNames.